client/task-list/tasks/payments/index.js

```diff
@@ -363,6 +370,7 @@ export default compose(
 			'woocommerce_bacs_accounts',
 			'woocommerce_eway_settings',
 			'woocommerce_razorpay_settings',
+			'woocommerce_mollie_settings',
```

Comment:
In `Methods.js` you are using `woocommerce_mollie_payments_settings`?

Reply:
I also added the setting here for pre-loading: https://github.com/woocommerce/woocommerce-admin/blob/main/src/Features/Onboarding.php#L708 I suppose not a must?

Reply:
Quite happy to do that I'll take a look